falcon/request.py

```diff
@@ -1160,6 +1162,8 @@ class RequestOptions(object):
             request's content type is
             *application/x-www-form-urlencoded* (default ``False``). In
             this case, the request's content stream will be left at EOF.
+        auto_parse_query_string_lists: Set to ``False`` in order to disable
```

Comment:
That makes sense. Perhaps you could just say something about how, for example, it can be used to make Falcon compatible with the style used by some front-end JS frameworks.